book/tables/efficacy/rbmit01.qmd

```diff
@@ -7,7 +7,7 @@ subtitle: Tables for RBMI
 
 {{< include ../../_utils/envir_hook.qmd >}}
 
-:::: {.panel-tabset}
+::: panel-tabset
```

Comment:
I will suggest to revert these colon changes. These are not failing the workflow but it would destroy the whole layout